k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml

```diff
@@ -223,3 +223,4 @@
     "sha256:aa079daa7efd93aa830e26483a49a6343354518360929494bad1d0ad3303142e": ["v20220906-g981ce38a7"]
     "sha256:41076fd9fb4255677c1a3da1ac3fc41477f06eba3c7ebf37ffc8f734dad51d7c": ["v20221114-controller-v1.5.1-6-ga66ee73c5"]
     "sha256:fa3f47c289236b104a1f5195b020657a29a3a9b0a355f86f2d52d42d1fa6495e": ["v20221220-controller-v1.5.1-53-g81d40b70a"]
+    "sha256:331b9bebd6acfcd2d3048abbdd86555f5be76b7e3d0b5af4300b04235c6056c9": ["v20230107-helm-chart-4.4.2-2-g96b3d2165"]
```

Comment:
We need to delete the old/rolled back 4.2.2 tag in ingress, and that will require a rebuild here.